api/workAllocation2/routes.ts

```diff
@@ -22,6 +23,7 @@ import { postFindPersonSearch } from './personService';
 const router = Router({ mergeParams: true });
 
 router.use(authInterceptor);
+router.delete('/caserole/:caseId/role/:roleId', deleteRoleByCaseAndRoleId);
```

Comment:
could this put to the roleAccess api folder?

Reply:
on \api\roleAccess\routes, there has been existing

router.post('/allocate-role/confirm', confirmAllocateRole);

It could be better to put all those role access api togeter here.

So it could be added with:
router.delete('/allocate-role/delete', deleteRoleByCaseAndRoleId);

It doesn't need to put the :caseId or :roleId in the path, it can be put in the delete payload body.

Reply:
Done